src/components/CodeBlock.tsx

```diff
@@ -21,13 +24,28 @@ const StyledPre = styled('pre', {
   mb: '$4',
   mt: 0,
   mx: '-$4',
-  overflow: 'hidden'
+  overflow: 'hidden',
+  position: 'relative',
+  '&:focus-within': {
+    boxShadow: '0 0 0 3px $primary500'
+  }
 })
 const StyledLivePreview = styled(LivePreview, {
   mx: '-$4',
   overflow: 'hidden',
   px: '$4',
-  py: '$5'
+  py: '$5',
+  variants: {
+    layout: {
```

Comment:
How come we only have one variant here? (just for my understanding)

Reply:
This is used just as an additional style without a default, by using the variants API from stitches. If you pass in `layout="center"` then you apply the style.

Reply:
I guess I'm wondering if we have a use case where we don't need it to be centered, but this adds more flexibility